src/Gate.java

```diff
@@ -7,12 +7,16 @@
  */
 import org.simgrid.msg.HostFailureException;
 import org.simgrid.msg.Msg;
+
+import java.util.Vector;
+
 import org.simgrid.msg.Host;
 import org.simgrid.msg.Process;
 import org.simgrid.msg.MsgException;
 
 public class Gate extends Job {
 
+	private String[] localOutput = {"/scratch/gate.sh.tar.gz", "/scratch/gate_release.tar.gz", "/scratch/file.zip"};
```

Comment:
This array is common to every Gate job. It should be a global variable (and a Vector) in VIPSimulator.java. For instance;
public static Vector<String> gateInputLocalNames = new Vector<String>("/scratch/gate.sh.tar.gz", "/scratch/gate_release.tar.gz", "/scratch/file.zip");